### PR TITLE
Bug/kanosdialog 1629

### PR DESCRIPTION
**Pull request by wizofe**
Kanos OK Button back to focus
Changed the KanosDialog class initialisation.

### CODE REVIEW

**Comment by wizofe** [comment]
[[P4] Password prompt in Kano Apps does not validate when pressing ENTER](https://trello.com/c/uStLZaqA/1629-p4-password-prompt-in-kano-apps-does-not-validate-when-pressing-enter)

File: .gitignore [hunk 1]
```diff
@@ -8,42 +8,7 @@ kano-splash/kano-splash
...
-*.lai
-*.la
-*.a
-*.lib
-
-# Executables
-*.exe
-*.out
-*.app
+# Created by https://www.gitignore.io/api/vim,macos,emacs,python,sublimetext,visualstudiocode
```
Comment (tombettany) [inline .gitignore]: We've lost `c++` along the way and we have C++ code in this repo.

**Comment by wizofe** [comment]
Done.

**Comment by tombettany** [comment]
Congratulations @wizofe - first code merged in!

**Pull request merged**